### DISCUSSION TITLE
Codegen fails using es5 (nextjs default)

### DISCUSSION THREAD

**Discussion by IHIutch** [chakra-ui/panda#850] [General]
### Description

I expected to start a new nextjs project using pandacss. However, when trying to add a custom recipe and using nextjs defaults (as of v13.4.7), `panda codegen` fails

`Transforming const to the configured target environment ("es5") is not supported yet`

### Link to Reproduction

https://stackblitz.com/edit/stackblitz-starters-wxqtbw?file=package.json

### Steps to reproduce

1. Open Link
2. Run `pnpm prepare`

### JS Framework

NextJS

### Panda CSS Version

0.5.0

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

### Additional Information

I'm able to bump to es6 and everything works fine. Im not sure which es versions are supported or should be supported, but I'm wondering if this just needs to be mentioned in the NextJS setup docs or if its an actual bug.

**Comment by segunadebayo** [comment]
I'll add a note to the nextjs docs. The target should be set to >=es5
